pkgs/shells/powershell/default.nix

```diff
@@ -14,8 +14,8 @@ let archString = if stdenv.isAarch64 then "arm64"
     platformLdLibraryPath = if stdenv.isDarwin then "DYLD_FALLBACK_LIBRARY_PATH"
                      else if stdenv.isLinux then "LD_LIBRARY_PATH"
                      else throw "unsupported platform";
-                     libraries = [ libunwind libuuid icu curl openssl_1_1 ] ++
-                       (if stdenv.isLinux then [ pam lttng-ust ] else [ darwin.Libsystem ]);
+    libraries = [ libunwind libuuid icu curl openssl_1_1 ] ++
+                  (if stdenv.isLinux then [ pam lttng-ust keyutils ] else [ darwin.Libsystem ]);
```

Comment:
```suggestion
    libraries = [ libunwind libuuid icu curl openssl_1_1 ]
      ++ lib.optionals stdenv.isLinux [ pam lttng-ust keyutils ]
      ++ lib.optionals [ darwin.Libsystem ];
```

Reply:
Thanks, I made the rest of the indentation less extreme as well.